quiz.js

```diff
@@ -0,0 +1,59 @@
+var heightInput = document.getElementById("height");
+var characterInput = document.getElementById("character");
+var submit = document.getElementById("submit");
+
+heightInput.addEventListener("input", treeFunction);
+characterInput.addEventListener("input", alertFunction);
+submit.addEventListener("click", submitFunction);
+
+
+function treeFunction() {
+  var height = heightInput.value;
+  var character = characterInput.value;
+  if (heightInput.value === "") {
+    alert("height is empty!")
+  }
+}
+
+function alertFunction() {
+ if (characterInput.value === "") {
+    alert("character is empty!")
+  }
+}
+
+
+function actualTreeFunction() {
+ var rows = document.getElementById("height").value;
+ var chars = document.getElementById("character").value;
```

Comment:
Lines 26 and 27 are repeating 11 and 12. When you combine the functions, see if you can eliminate two of the lines.